packages/bpk-component-dialog/src/common-types.ts

```diff
@@ -26,11 +26,12 @@ export const HEADER_ICON_TYPES = {
 
 export type DialogInnerProps = {
   ariaLabel: string;
+  ariaModal: boolean;
```

Comment:
You will need to make sure you add this to the `common-types.d.ts` file so that it will come through in the TS types otherwise it will cause issues